src/ssWeatherManager.lua

```diff
@@ -155,6 +155,10 @@ function ssWeatherManager:readStream(streamId, connection)
 
         table.insert(self.rains, rain)
     end
+
+    -- we need to synch this, we have an error log entry on dedicated server otherwise
+    -- for now i do it here, it's not elegant but at the moment it work
+    ssSnow.appliedSnowDepth = streamReadFloat32(streamId)

```

Comment:
Ok, yes, you are right, but than the complete function ssSnow:hourChanged should only be run on the server and not run on the client otherwise we became an error while playing on a dedicated server for the client, server runs fine.